components/admin_console/team_channel_settings/channel/details/channel_profile.tsx

```diff
@@ -26,7 +44,7 @@ export const ChannelProfile: React.SFC<ChannelProfileProps> = (props: ChannelPro
             subtitleId={t('admin.channel_settings.channel_detail.profileDescription')}
             subtitleDefault='Summary of the channel, including the channel name.'
         >
-            <div className='group-teams-and-channels'>
+            <div className='group-teams-and-channels AdminChannelDetails'>
```

Comment:
do we need to keep the old class?

Reply:
I'm not sure how to deal with this. My thinking was to use the new naming convention, but there are still a bunch of styles tied to `.group-teams-and-channels`. What do you think? We could:

a) replace `AdminChannelDetails` with `group-teams-and-channels` and tie the new styles to it instead
c) update `group-teams-and-channels` to be `GroupTeamsAndChannels` and then all of the associated states like `group-teams-and-channels--body`, etc, this does mean updating a bunch of things including tests, not insurmountable just noting it
d) new stuff goes under `AdminChannelDetails` (ie leave it as-is in the PR)

Thoughts? Other ideas?